### PR TITLE
[UT] Fix unstable ut

### PR DESCRIPTION
**Pull request by Youngwb**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Fix unstable UT whcih MVActiveChecker caused it

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by Youngwb** [comment]
run all

**Comment by Youngwb** [comment]
run all

File: fe/fe-core/src/test/java/com/starrocks/utframe/UtFrameUtils.java [hunk 1]
```diff
@@ -235,7 +236,10 @@ public static synchronized void createMinStarRocksCluster(boolean startBDB) {
             ClientPool.heartbeatPool = new MockGenericPool.HeatBeatPool("heartbeat");
             ClientPool.backendPool = new MockGenericPool.BackendThriftPool("backend");
 
+            FeConstants.runningUnitTest = true;
             startFEServer("fe/mocked/test/" + UUID.randomUUID().toString() + "/", startBDB);
+            FeConstants.runningUnitTest = false;
```
Comment (ABingHuang) [inline fe/fe-core/src/test/java/com/starrocks/utframe/UtFrameUtils.java:241]: why set runningUnitTest like this?

**Review by Seaven — DISMISSED** [review]

**Review by ABingHuang — APPROVED** [review]

**Comment by Youngwb** [comment]
run starrocks_fe_unittest

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVActiveChecker.java|3|3|100.00%|[]|

**Comment by Youngwb** [comment]
run starrocks_admit_test

**Comment by Youngwb** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**